pennylane/devices/qutrit_mixed/initialize_state.py

```diff
@@ -0,0 +1,102 @@
+"""Functions to prepare a qutrit mixed state."""
+
+from typing import Iterable, Union
+import numpy as np
+import pennylane as qml
+from pennylane import (
+    QutritBasisState,
+)
+from pennylane.operation import (
+    StatePrepBase,
+    Operation,
+)
+from pennylane.wires import Wires
+
+qudit_dim = 3  # specifies qudit dimension
+
+
+def create_initial_state(
+    wires: Union[qml.wires.Wires, Iterable],
+    prep_operation: Operation = None,
+    like: str = None,
+):
+    r"""
+    Returns an initial state, defaulting to :math:`\ket{0}\bra{0}` if no state-prep operator is provided.
+
+    Args:
+        wires (Union[Wires, Iterable]): The wires to be present in the initial state
+        prep_operation (Optional[StatePrepBase]): An operation to prepare the initial state
+        like (Optional[str]): The machine learning interface used to create the initial state.
+            Defaults to None
+
+    Returns:
+        array: The initial state of a circuit
+    """
+    if isinstance(wires, Wires):
+        wire_array = wires.toarray()
+    else:
+        wire_array = np.array(wires)
```

Comment:
I'm a little curious as to why we want to have a numpy array of the wire labels, instead of a `Wires` object or just a plain list/tuple.  Wires aren't really numerical data. They're more like metadata that controls what the actual data does.

Could we leaves `wires` as `Wires` objects?

Reply:
Maybe the else isn't necessary I just did it for consistency and to change the name, however I realize that it is unnecessary.

The wires.toarray() was to be consistent with the new default.qubit, I will look into keeping wires consistent.

Reply:
I updated this to remove the else part, I turned the wires to a list instead, which it is used as later to find the length and calculate some values. Is this an okay change @albi3ro?

Reply:
I changed to remove this, length is the only necessary thing due to Olivia's suggestion